tests/integ/modin/frame/test_applymap.py

```diff
@@ -107,7 +107,7 @@ def test_applymap_numpy(func):
     native_df = native_pd.DataFrame(data)
     snow_df = pd.DataFrame(data)
 
-    with SqlCounter(query_count=7, udf_count=1):
+    with SqlCounter(query_count=1):
```

Comment:
This is great

Reply:
qq: what change has reduce the query_count from 7 to 1 and eliminate the udf call?